winservices-config.yml.sample

```diff
@@ -1,7 +1,5 @@
 integrations:
-  - name: com.newrelic.winservices
-    exec:
-      - C:\Program Files\New Relic\newrelic-infra\newrelic-integrations\nri-winservices.exe
+  - name: nri-winservices
```

Comment:
for more context look [this](https://docs.newrelic.com/docs/create-integrations/infrastructure-integrations-sdk/specifications/host-integrations-newer-configuration-format#exec)